server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionStatusManager.java

```diff
@@ -199,21 +197,13 @@ private class CheckTask implements Callable<ConnectionTestResult> {
         final CheckState checkState;
         final User user;
 
-        CheckTask(ConnectionConfig connection, CheckState checkState, ConnectionAccountType accountType) {
-            this.testConnectionReq = TestConnectionReq.fromConnection(connection, accountType);
+        CheckTask(ConnectionConfig connection, CheckState checkState) {
+            this.testConnectionReq = TestConnectionReq.fromConnection(connection, ConnectionAccountType.MAIN);
```

Comment:
`ConnectionAccountType.MAIN` should be deleted?

Reply:
no, this code is also used at `DataTransferService#testSysTenantAccount`